cmd/krel/cmd/root.go

```diff
@@ -60,8 +62,19 @@ func Execute() {
 }
 
 func init() {
-	rootCmd.PersistentFlags().BoolVar(&rootOpts.nomock, "nomock", false, "nomock flag")
-	rootCmd.PersistentFlags().StringVar(&rootOpts.logLevel, "log-level", "info", "the logging verbosity, either 'panic', 'fatal', 'error', 'warn', 'warning', 'info', 'debug' or 'trace'")
+	rootCmd.PersistentFlags().BoolVar(
+		&rootOpts.nomock,
+		"nomock",
+		false,
+		"run the command to target the production environment",
```

Comment:
nitpick: should be consistent between this description and the other `nomock` description in this PR?

Reply:
I was intending to leave this more general, because what "nomock" means is kinda dependent on many factors (git is using dry run, artifact push is using different bucket).

Reply:
Gotcha, that seems reasonable 👍